.go-version

```diff
@@ -1 +1 @@
-1.12.9
+1.12.17
```

Comment:
Given we're on Go 1.14 now, should we bump the version accordingly?

Reply:
I think in general I held off go1.13 personally (for all my development) for some weird transitive dependency things... I can't remember them at all so sure let's move to go1.14!